src/pages/ApplyPage.css

```diff
@@ -165,8 +165,8 @@ html {
 @supports (background-blend-mode: multiply) {
   .applyPage__showcase {
     background-image: linear-gradient(
-      var(--primary-green, rgb(106, 235, 80)),
-      rgb(89, 201, 209)
+      var(--primary-green, rgb(142, 236, 130)),
+      rgb(100, 157, 211)
```

Comment:
The colour gradient looks good! I would just suggest to add these new colors as variables in App.css and referencing them here.